vm/src/vm/vm_new.rs

```diff
@@ -270,7 +270,11 @@ impl VirtualMachine {
         } else {
             Vec::new()
         };
-        self.new_exception(self.ctx.exceptions.stop_iteration.to_owned(), args)
+        let exc = self.new_exception(self.ctx.exceptions.stop_iteration.to_owned(), args.clone());
+        exc.as_object()
+            .set_attr("value", self.unwrap_or_none(args.get(0).cloned()), self)
+            .unwrap();
```

Comment:
We dont use unwrap for python errors. Try ? operator please

Reply:
Thank you for comments!
I tried to use `?` operator behind `set_attr()`, but it didn't work.
I got an error message. 
"The return type is not implemented for `core::PyRef<exceptions::types::PyBaseException>`"

Reply:
```rust
match name_error.as_object().set_attr("name", name, self) {
    Ok(_) => name_error,
    Err(e) => e,
}
```
We can avoid `unwrap` like this because `Err` contains PyRef<PyBaseException>`.
But it seems there are several methods use `unwrap`. Maybe we can make some inline method for this kind of actions

Reply:
Thank you!
I updated the code and It works.